src/test/java/org/cactoos/http/io/SkipInputTest.java

```diff
@@ -26,10 +26,10 @@
 
 import org.cactoos.io.BytesOf;
 import org.cactoos.io.InputOf;
+import org.cactoos.matchers.TextHasString;
```

Comment:
@Vatavuk please use `org.llorllale.cactoos.matchers.TextHasString` (we'll soon be removing `org.cactoos.matchers` from the classpath)